akka-testkit/src/test/scala/akka/testkit/TestExplicitlyTriggeredScheduler.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2020-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.testkit
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import scala.concurrent.duration.DurationInt
+
+class TestExplicitlyTriggeredScheduler extends AkkaSpec {
```

Comment:
Our convention is to name this more like `ExplicitlyTriggeredSchedulerSpec`

Reply:
Renamed